docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,27 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | True if female                              | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`pflichtbeitragszeit`  | Months of mandtory contributions            | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`freiw_beitragszeit`   | Months of voluntary contributions           | float        |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit`     || Months of sickness, pregnancy/Mutterschutz,|| float       |
+||                        || unemployment, school/university after age  ||             |
+||                        || 17 (max 8 years), insurance retirement     ||             |
++-------------------------+---------------------------------------------+--------------+
+|| _`anrechnungszeit_45`  || Months of sickness, unemployment (only time|| float       |
+||                        || of Entgeltersatzleistungen, not ALGII),    ||             |
+||                        || interim payments (Übergangsgeld), marginal ||             |
+||                        || employment                                 ||             |
++-------------------------+---------------------------------------------+--------------+
+|| _`ersatzzeit`          || Months during military, persecution/escape,|| float       |
+||                        || internment and consecutive sickness        ||             |
++-------------------------+---------------------------------------------+--------------+
+| _`kinder_berückz`       | Months of childcare till age 10             | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`pflege9295_berückz`   | Months of home care (01.01.1992-31.03.1995) | float        |
++-------------------------+---------------------------------------------+--------------+
+| _`jahre_beiträg_nach40` | Years of mandat. contributions after age 40 | float        |
```

Comment:
```suggestion
| _`jahre_beiträge_nach40` | Years of mandatory contributions after age 40 | float        |
```
if too long, abbreviate to `beitr`

Also, please make sure that at least in all descriptions it becomes clear that we are talking about public pensions. Ideally, we would even do so in the variable names but that will not work given the restrictions we have, I am afraid.

Reply:
changed